deploy/crds/infinispan.org_infinispans_crd.yaml

```diff
@@ -810,6 +810,25 @@ spec:
                 - type
                 type: object
               type: array
+            podStatus:
+              properties:
+                ready:
+                  description: Deployments are ready to serve requests
+                  items:
+                    type: string
+                  type: array
+                starting:
+                  description: Deployments are starting, may or may not succeed
+                  items:
+                    type: string
+                  type: array
+                stopped:
+                  description: Deployments are not starting, unclear what next step
```

Comment:
@dmvolod Does this mean "not able to start" due to some fault? When would this description appear to users?

Reply:
@oraNod this is auto-generated values from the RedHat syseng operator-utils component. We can't to update values in our code